src/components/AddBookForm.js

```diff
@@ -59,104 +66,131 @@ export default function AddBookForm({
+                placeholder="978123456789"
+                defaultValue={searchedBook.isbn13 || searchedBook.isbn10}
+                maxLength="13"
+                pattern="[A-Za-z0-9]+"
+                autoComplete="off"
+                required
+              />
+            </>
           )}
+          <label htmlFor="bookTitle">Book title:</label>
           <input
-            type="file"
-            name="bookcover"
-            id="bookcover"
-            accept=".png, .jpeg, .jpg"
-            onChange={event => {
-              getBookcoverPreview(event)
-            }}
-          />
-          <label aria-label="select bookcover" htmlFor="bookcover">
-            Select
-          </label>
-        </BookcoverContainer>
-        <div>
-          <label htmlFor="reading-since">Reading since:</label>
-          <input
-            name="readingSince"
-            type="date"
-            id="reading-since"
-            max={getToday()}
+            name="title"
+            type="text"
+            id="bookTitle"
+            placeholder="Title of the book you want to add"
+            defaultValue={searchedBook ? searchedBook.title : ''}
+            autoComplete="off"
+            required
           />
-          <label htmlFor="onPage">Currently on page:</label>
+          <label htmlFor="bookAuthors">Author or Authors:</label>
           <input
-            name="onPage"
-            type="number"
-            id="onPage"
-            placeholder="e. g. 72"
+            name="authors"
+            type="text"
+            id="bookAuthors"
+            placeholder="Name or names of the author/authors"
+            defaultValue={
+              searchedBook ? handleAuthorsLength(searchedBook.authors) : ''
+            }
             autoComplete="off"
+            required
           />
-        </div>
-      </OptionalContentContainer>
-      <button>Add book to list</button>
+        </MainContentContainer>
+        <OptionalContentContainer>
+          <BookcoverContainer>
+            {searchedBook ? (
+              <div id="bookcover-preview">
+                <img src={searchedBook.thumbnail} alt="bookcover" />
+              </div>
+            ) : (
+              <div id="bookcover-preview">
+                {bookcover === placeholder ? (
+                  <img src={previewPlaceholder} alt="bookcover" />
+                ) : (
+                  <img src={bookcover} alt="bookcover" />
+                )}
+              </div>
+            )}
+            <input
+              type="file"
+              name="bookcover"
+              id="bookcover"
+              accept=".png, .jpeg, .jpg"
+              onChange={event => {
+                getBookcoverPreview(event)
+              }}
+            />
+            <label aria-label="select bookcover" htmlFor="bookcover">
+              Select
+            </label>
+          </BookcoverContainer>
+          <ReadingStatus>
+            {!isFinished ? (
```

Comment:
here, too, even so I can understand that you choosed it this way, because it is set as `false` by default.